### PR TITLE
Make pytest output less verbose

### PR DESCRIPTION
**Pull request by gitanshu**
https://docs.pytest.org/en/latest/capture.html

### CODE REVIEW

**Comment by ryanjulian** [comment]
I enabled this flag intentionally, to (1) make the test output less chatty (it's already over 5k lines, which is a problem) and (2) prevent people from writing tests which rely on stdout (e.g. print() testing) on the CI.

Is there a reason you need this disabled? I was already contemplating making the current test output *less* verbose by removing the pass/fail line for each test.

**Comment by ryanjulian** [comment]
These look much better -- maybe we can also skip printing the coverage report?

It's not super useful because we have multiple test jobs.

**Comment by ryanjulian** [comment]
Sorry to keep piling on here -- did we change some flag about warning reporting?

Our warning reports don't show which test cases have warnings, which makes it difficult to resolve them when they come from general framework code.

The example shows each warning associated with a test: https://docs.pytest.org/en/latest/warnings.html

**Comment by gitanshu** [comment]
> These look much better -- maybe we can also skip printing the coverage report?
Looking into this

> did we change some flag about warning reporting?
looks like this has always been the behavior for us. We don't change anything regarding warnings anywhere though :thinking:

**Comment by gitanshu** [comment]
@ryanjulian this is warnings summary without xdist: https://travis-ci.com/github/rlworkgroup/garage/jobs/321427488#L1444
VS
this is with xdist
https://travis-ci.com/github/rlworkgroup/garage/jobs/321419199#L1320

> Our warning reports don't show which test cases have warnings 

Seems like warnings not showing the test name is a side effect of pytest-xdist :crying_cat_face:

File: Makefile [hunk 1]
```diff
@@ -27,24 +27,23 @@ ci-job-precommit: assert-docker docs
 
 ci-job-normal: assert-docker
 	[ ! -f $(MJKEY_PATH) ] || mv $(MJKEY_PATH) $(MJKEY_PATH).bak
-	pytest --cov=garage -v -m \
-	    'not nightly and not huge and not benchmark and not flaky and not large and not mujoco' --durations=0
+	pytest --cov=garage --cov-report=xml -m \
+	    'not nightly and not huge and not benchmark and not flaky and not large and not mujoco' --durations=20
 	coverage xml
```
Comment (gitanshu) [inline Makefile:32]: Remove this

File: Makefile [hunk 2]
```diff
@@ -27,24 +27,23 @@ ci-job-precommit: assert-docker docs
...
 	bash <(curl -s https://codecov.io/bash)
 
 ci-job-large: assert-docker
 	[ ! -f $(MJKEY_PATH) ] || mv $(MJKEY_PATH) $(MJKEY_PATH).bak
-	pytest --cov=garage -v -m 'large and not mujoco' --durations=0
+	pytest --cov=garage --cov-report=xml -m 'large and not mujoco' --durations=20
 	coverage xml
```
Comment (gitanshu) [inline Makefile:38]: Remove this

**Pull request merged**